src/main/kotlin/step2/Calculator.kt

```diff
@@ -0,0 +1,43 @@
+package step2
+
+import step2.ExceptionType.Companion.INPUT_MUST_END_WITH_NUMBER
+import step2.ExceptionType.Companion.INPUT_MUST_NOT_BLANK
+import step2.ExceptionType.Companion.INPUT_MUST_NOT_NULL
+import step2.ExceptionType.Companion.INPUT_MUST_START_WITH_NUMBER
+import step2.ExceptionType.Companion.NUMBER_NEXT_IS_MUST_OPERATOR
+import step2.ExceptionType.Companion.OPERATOR_NEXT_IS_MUST_NUMBER
+
+object Calculator {
+    private fun String.isNumeric() = matches("-?\\d+(\\.\\d+)?".toRegex())
+    private fun String.isNotNumeric() = !isNumeric()
+    private fun String.getOperator() = OperatorType.getOperator(this)
+
+    fun calculate(input: String?): Double {
+        val checkedInput = checkInputValidation(input)
+        val dividedList = checkedInput.split(" ")
+        checkElementValidation(dividedList)
+
+        var idx = 0
+        var accumulator = dividedList.first().toDouble()
+
+        while (idx < dividedList.size) {
```

Comment:
```suggestion
        repeat(dividedList.size) {
```
while 대신 repeat을 사용하시면 idx라는 변수를 사용하지 않아도 됩니다.
가급적이면 var을 사용하지 않으려는 습관을 들이시면 좋습니다 😄